app/views/download/help_improve.html.haml

```diff
@@ -85,8 +81,164 @@
+
+          errorFn.clearErrorWarnings(form)
 
+          var isGov = readFormFn.checkAnswer(form, 'help_us_improve_user[is_government]')
+
+          switch (isGov) {
+            case 'Yes':
+              // check questions
+              var partOfGov = readFormFn.checkAnswer(form, 'help_us_improve_user[gov_what_part_of_government]')
+              var registerUse = readFormFn.checkAnswer(form, 'help_us_improve_user[gov_what_are_you_using_registers_for]')
+
+              if (partOfGov === false) {
+                state.errorSummary.push({
+                  messageShouldRead: 'Enter a government organisation',
+                  link: '#which-part-of-government'
+                })
+                errorFn.showErrorWarning(form, {
+                  addClassTo: '.js--which-part-of-government-group',
+                  putMessageIn: '.js--which-part-of-government-message',
+                  messageShouldRead:  'Enter a government organisation'
+                })
+              }
+
+              if (registerUse === false) {
+                state.errorSummary.push({
+                  messageShouldRead: "Select what you're using registers for",
+                  link: '#gov-registers-use'
+                })
+                errorFn.showErrorWarning(form, {
+                  addClassTo: '.js--gov-registers-use-group',
+                  putMessageIn: '.js--gov-registers-use-message',
+                  messageShouldRead:  "Select what you're using registers for"
+                })
+              }
+
+              if (registerUse === false || partOfGov === false) {
+                errorFn.showErrorSummary(form)
+              }
+              else {
+                readFormFn.sendData(form);
+                form.submit();
+              }
+              break
+            case 'No':
+              var registerUse = readFormFn.checkAnswer(form, 'help_us_improve_user[nongov_what_are_you_using_registers_for]')
+              if (registerUse === false) {
+                state.errorSummary.push({
+                  messageShouldRead: "Select what you're using registers for",
+                  link: '#nongov-registers-use'
+                })
+                errorFn.showErrorWarning(form, {
+                  addClassTo: '.js--nongov-registers-use-group',
+                  putMessageIn: '.js--nongov-registers-use-message',
+                  messageShouldRead: "Select what you're using registers for"
+                })
+                errorFn.showErrorSummary(form)
+              }
+              else {
+                readFormFn.sendData(form);
+                form.submit();
+              }
+              break
+            case false:
+            default:
+              errorFn.showErrorWarning(form, {
+                addClassTo: '.js--is-government-group',
+                putMessageIn: '.js--is-government-message',
+                messageShouldRead: 'Select yes if you work for the government'
```

Comment:
I think this should be _Select if you work for the government_ rather than leaning towards one answer.

Reply:
I've moved the error messages to the i18n config file. 

The wording follows the guidelines in the design system and Mark has okayed it.